tests/SprykerSdkTest/Zed/AppSdk/Business/AppManifestValidatorFacadeTest.php

```diff
@@ -100,16 +100,14 @@ public function testValidateManifestReturnsFailedResponseWhenPageBlockTypeFieldH
      */
     public function testValidateManifestReturnsFailedResponseWhenFilesNotFound(): void
     {
-        // Act
+       // Act
```

Comment:
Revert please.

Reply:
Can you please make sure that all comments from me are either accepted and applied or commented?

Reply:
It's still visible, please push your code changes.